packages/frontend/src/pages/ProjectSettings.tsx

```diff
@@ -25,12 +29,19 @@ import {
 const ProjectSettings: FC = () => {
     const { projectUuid } = useParams<{ projectUuid: string }>();
     const { isLoading, data, error } = useProject(projectUuid);
+    const { user } = useApp();
 
+    const [projectAccessCreate, setProjectAccessCreate] =
+        useState<boolean>(false);
     const basePath = useMemo(
         () => `/projects/${projectUuid}/settings`,
         [projectUuid],
     );
 
+    const isOrgAdmin = user.data?.ability.can(
```

Comment:
Since the backend doesn't implement any access controls for these endpoints, I didn't expect this here, are you going to update the backend too with the matching abilities?

Reply:
This is the only ability I have implemented in this PR, this is so editors can't see this page, like the title in the settings suggests: `only visible to org admins`

All other ability checks will be implmented on a separate PR (I feel this one is big enough) . There are some TODOs for those